.github/workflows/pr-merge.yml

```
@@ -0,0 +1,56 @@
+  pull_request:
+    types: [closed]
+    branches: [main, conda]
+
+jobs:
+  notify-discord-merge:
+    name: Notify Discord (merged PR)
+    runs-on: ubuntu-20.04
+    if: github.event.pull_request.merged == true
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)